BUILD.md

```diff
@@ -33,6 +37,8 @@ On Windows execute `gradlew.bat assembleDebug`.
 
 If you want to build and install on a connected device or emulator image, run ```gradle installDebug``` instead.
 
+If you want to build it through [Android Studio](https://developer.android.com/studio), open project ```android\build.gradle``` in Android Studio.
```

Comment:
That's not 100% correct. At least on Windows you just select the android folder:

![image](https://github.com/SaschaWillems/Vulkan/assets/10283127/3b42b781-89e5-4c79-bdd4-a55c28f32b0f)

Reply:
Change to just select the android folder.